VirtoCommerce.Domain/Pricing/Model/Search/PricesSearchCriteria.cs

```diff
@@ -48,5 +48,7 @@ public string[] ProductIds
                 _productIds = value;
             }
         }
+
+        public DateTime? Modified { get; set; }
```

Comment:
I think to use `Modified ` property name not so obvious because it is not clear which records will be retrieved, more than this date or less or equal

Reply:
I see what you mean. I could create a ModifiedPredicate that holds a datetime and an operator 'before', 'after', 'on'. How would you like that?

Reply:
I think renaming to `ModifiedSince` will be enough

Reply:
Thanks @tatarincev , I'll do it in a sec.